meson.build

```diff
@@ -112,7 +112,8 @@ if not cc.links('', name: '-Wl,--version-script', args: ['-shared', version_link
   version_link_args = []
 endif
 
-cython = find_program('cython')
+# generator() doesn't accept compilers, only found programs. Cast it.
+cython = find_program(meson.get_compiler('cython').cmd_array()[0])
```

Comment:
This casts back to a string, typically also `'cython'`. So this changes nothing, except for possibly picking up the workaround for Debian, in case only a `cython3` and no `cython` executable is present on the machine. Right?

Reply:
In principle the path of the compiler binary can be defined in a native-build.ini file, cf. https://mesonbuild.com/Machine-files.html#binaries

Reply:
Yes, that's true. But ideally that should not be needed - and if it is, it should only be for things that are really nonstandard and that the average user will never run into.

Reply:
> So this changes nothing, except for possibly picking up the workaround for Debian, in case only a `cython3` and no `cython` executable is present on the machine. Right?

Correct.